auth/token.go

```diff
@@ -84,16 +85,15 @@ func (t stdToken) IssuedAt() time.Time {
 }
 
 func (t stdToken) Issuer() string {
-	// support ias custom domains
-	iss := t.IasIssuer()
-	if iss == "" {
-		iss = t.jwtToken.Issuer()
-	}
-	return iss
+	return t.jwtToken.Issuer()
```

Comment:
I personally would prefer it the other way around:
- `Issuer` shall always return the IAS Issuer, ias_iss or if not exists ias
- `IasIssuer` i would deprecate but return `t.Issuer` for now
- `CustomIssuer` shall return ias (in case ias_iss is given) otherwise null.

Reply:
Hi @f-blass 
i would prefer to have our client libs implementations synchronised. I have already aligned with Stefan the other way, but I will double check the recording if you like.

Reply:
Btw. I can take over this PR if you like.

Reply:
I changed the accessibility for most claims to private to match the Java implementation. They are no longer usable from outside of the package.

Reply:
Hi @f-blass you can check the claims exposed by Java here:
https://github.com/SAP/cloud-security-xsuaa-integration/blob/main/java-api/src/main/java/com/sap/cloud/security/token/Token.java